string-manipulation-1/string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,17 @@
+/* exported isLowerCased */
+function isLowerCased(string) {
+  const lowerCasedString = string.toLowerCase();
+  let count = 0;
+
+  for (let i = 0; i < string.length; i++) {
+    if (lowerCasedString[i] === string[i]) {
+      count++;
+    }
+  }
+
+  if (count === lowerCasedString.length) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
We can actually simplify this a lot! We don't even need to use a `for` loop. We can convert the whole string to lowercase and compare it to the original string
```
function isLowerCased(word) {
  return word === word.toLowerCase();
}
```